### PR TITLE
Update Android Gradle to 7.0.0

### PR DESCRIPTION
**Pull request by bryanoostra**

### CODE REVIEW

**Comment by nhaarman** [comment]
This is not gradle, but the Android Gradle Plugin 🙈 
Gradle is currently at version 6.8

**Comment by nhaarman** [comment]
To fix CI, use `actions/setup-java`:

```
name: Test

on: pull_request

jobs:
  test:
    runs-on: ubuntu-latest

    steps:
      - name: Checkout
        uses: actions/checkout@v2

      - name: Set up Java 11
        uses: actions/setup-java@v1
        with:
          java-version: '11'

      - name: ktlint
        run: |
          curl -sSLO https://github.com/pinterest/ktlint/releases/download/0.39.0/ktlint
          chmod a+x ktlint
          ./ktlint "**/src/**/*.kt" --color --relative

      - name: Test
        run: ./gradlew test
```

File: dependencies.gradle [hunk 1]
```diff
@@ -1,11 +1,11 @@
-ext.composeVersion = "1.0.0-alpha07"
+ext.composeVersion = "1.0.0-alpha09"
 
 dependencyManagement {
     dependencies {
         dependency "androidx.appcompat:appcompat:1.3.0-alpha02"
         dependency "androidx.compose.material:material:$composeVersion"
         dependency "androidx.compose.ui:ui:$composeVersion"
-        dependency "androidx.ui:ui-tooling:$composeVersion"
+        dependency "androidx.ui:ui-tooling:1.0.0-alpha07"
```
Comment (bryanoostra) [inline dependencies.gradle:8]: so I'm a little confused here:
The version >alpha7 aren't available
https://mvnrepository.com/artifact/androidx.ui/ui-tooling

the name seems to be changed to androidx.compse.ui:ui-tooling,
https://stackoverflow.com/questions/64961333/gradle-cant-resolve-androidx-uiui-tooling-dependency-by-google-repository

however this doesn't seem to contain the `@Preview` annotation ?

Comment (nhaarman) [reply dependencies.gradle:8]: Ah, the artifact is renamed. Use `dependency "androidx.compose.ui:ui-tooling:$composeVersion"`

**Pull request merged**